api/v1alpha1/drpolicy_types.go

```diff
@@ -20,13 +20,46 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// DR cluster information
+type DRClusterInfo struct {
+	// Name of this ManagedCluster as configured in OCM/ACM
+	ManagedClusterName string `json:"managedClusterName"`
+
+	// Specifies the name of the S3 profile (in RamenConfig) that is expected to
+	// survive when this DR cluster (with the given ManagedClusterName) is used
+	// to recover from a disaster.  The Ramen operators will upload PV related
+	// cluster state from other DR peers to this S3 store and will download from
+	// this store to this DR cluster when this DR cluster becomes the primary
+	// server of the application.  There are two broad types of S3 store
+	// configuration that are supported:  the S3 store could either be in the
+	// same failure domain (region or AZ) as this DR cluster or could be in a
+	// failure domain that is outside of all the failure domains of the DR
+	// cluster set.
+	// 1.In the failure domain of this DR cluster: When using S3 stores that are
+	// located in the same region as the DR cluster, such as, by using the
+	// Noobaa S3 service on the DR cluster, this S3 profile should point to the
+	// S3 store that is located in the same region as the DR cluster.  Other DR
+	// DR cluster in the DRClusterSet will upload their PV related cluster state
+	// to this DR cluster's S3 store to protect their application in case they
+	// fail.  When this DR cluster recovers the application, the PV related
+	// cluster state stored in this S3 store will be downloaded and applied to
+	// this DR cluster's etcd state.
+	// 2.Outside the failure domain of all DR clusters: When using an S3 store
+	// that is located in a failure domain that is unrelated to DR clusters in a
+	// given DRClusterSet, all DR clusters in that set can use the same S3
+	// profile that points to sucn a single S3 store that is used for both
+	// uploading and downloading PV related cluster state, as it will not be
+	// affected by failure of any DR clusters in that set.
+	S3ProfileName string `json:"s3ProfileName"`
```

Comment:
Multicluster Orchestrator doesn't have S3 information today. So can we make this optional?
This will unblock us until we can get S3 information.

Reply:
At the DRPolicy level, which is at the hub, you can set the S3 profile to "any" string. However, you may be able to workaround the S3 requirement by setting the S3 profile in [VolumeReplicationGroup](https://github.com/RamenDR/ramen/pull/117/files#diff-f268ac6aaecc3265edfcb32673b4bb4d2ef7048e26f941b4aa842a0da928aa5dR79) to an empty string, as it is optional.

Reply:
I will work with a dummy value (or an empty string if it allows). Thanks.